modules/serverless-rn-1-33-0.adoc

```diff
@@ -18,11 +18,13 @@
 * {ServerlessProductName} now uses Knative for Apache Kafka 1.12.
 * The `kn func` CLI plugin now uses `func` 1.14.
 
-* OpenShift Serverless on ARM64 is now available as Technology Preview.
+* {ServerlessProductName} on ARM64 is now available as Technology Preview.
 * The `NamespacedKafka` annotation is now deprecated. Use the standard Kafka broker without data plane isolation instead.
 
 * When enabling the automatic `EventType` auto-creation, you can now easily discover events available within the cluster. This functionality is available as a Developer Preview.
 
+* You can now explore the Knative Eventing monitoring dashboards directly within the *Observe* tab of the developer view in the OpenShift Developer Console.
```

Comment:
Should "developer view" be "Developer perspective"?

Reply:
It is the developer view.